### ISSUE TITLE
Vision for OpenDataPolicies.org

### ISSUE THREAD

**Issue by gregjd** [sunlightpolicy/opendatapoliciesstatic#80]
Vision for OpenDataPolicies.org
==========

OpenDataPolicies.org will serve as the single gateway for all of Sunlight's resources related to (local) open data — resources that are currently scattered across _Public Policy for Public Data_ (PP4PD), _OpenDataPolicies.org_, and numerous Google Drive files.

Its status as a one-stop shop for resources would make it a central piece of our What Works Cities "help desk" model, as well as an easy way to serve cities (and other governments/agencies) that are not part of the WWC program.

## Components

The site's components will include:

1. **Checklist:** This will be an ordered checklist for cities (or any government/agency) that are interested in open data and want to draft and implement a policy. It will be an updated version _Public Policy for Public Data_. (Any of our relevant resources that are currently in Google Docs could be moved to be subpages of the Checklist, if desired.)

2. **Collection:** This would be our expanding compendium of policies that already resides on the current site, but with semantically tagged text and a richer set of metadata.

3. **Guidelines:** This will be the new home of our Open Data Policy Guidelines. (Whether they're the current ones or an updated set is a separate concern.) Having our Open Data Policy Guidelines on this site will allow a seamless interconnection with the Collection of policies, including the ability to see matching text for specific guidelines — potentially eliminating the need for the policy-comparison spreadsheet.

4. **Generator:** This will help anyone who wants to write a policy do just that. It will be some sort of combination of the current Firestarter/Wizard and an export/remix tool for language found in the Collection of existing policies. The exact form of this resource is flexible.

5. **Analytics:** This can be a home for analytics products (broadly defined) that draw on the Collection of policies as a data source. This could include statistics on: the portion of cities of different sizes (and in different regions) that have open-data policies, the frequency of different legal means, and the adoption rates of our various guidelines. There could also be visualizations of any of that data, as well as a dashboard of metrics re: the What Works Cities program and WWC policies.

## Design

The details of the site's design obviously don't need to be addressed for a while, but it could either use the current _Public Policy for Public Data_ theme or a new one.

The important thing to note now is that the site should feel like "one resource with multiple component resources." As examples, see the following:

![Sample branding for resources on OpenDataPolicies.org](https://cloud.githubusercontent.com/assets/3474451/22664229/a90a872c-ec7d-11e6-8aa3-83350bac975e.png)

![Tennessee's new logos for state agencies](https://cloud.githubusercontent.com/assets/3474451/22664104/2a144476-ec7d-11e6-832f-ecfe04f5a589.png)

![KC&B logo examples](https://cloud.githubusercontent.com/assets/3474451/22664145/560daa22-ec7d-11e6-9cbc-5584bcf7a71a.png)

Read more:
- [KC&B: Building a Solid Foundation Through Branding](http://metricmarketing.ca/case-studies/kc-b/)
- [Tennessee's new logo](https://govfresh.com/2015/08/tennessees-new-logo/)

## How to get there

In short, I'd recommend splitting implementation into two phases, with the first phase involving the launch of the new site and the second phase involving the launch of the Generator plus semantic tagging and an updated set of Guidelines.

Many of these items are independent or can be executed in parallel. Of note, I've described many of these components as beefed-up versions of existing ones, so we could create a unified site before beefing up those components, or we could beef up first, or some combination. That said, creating the unified site is a great opportunity to draw additional attention to the site, so it would be _nice_ to have some beefing-up that can be announced at the same time.

There are some components that seem unlikely to be done within the next two months, including semantic tagging of policy text and an updated set of guidelines. So these and potentially other aspects could be part of a "2.0" version of the site; I think these would be enough exciting developments to warrant drawing "major update" attention for a second time.

Given that, phase one could involve creating the single site with a unified design/branding and the Checklist, Collection, Guidelines (current edition, with a link to the comparison Google Sheet). I suspect phase one could be done within two months, maybe even a month — it really depends on the details. Phase one would _not_ include the Generator, as that depends on semantic tagging. Instead, we can continue to link people to the Wizard.

The Analytics component could be added at any time; it's not essential to the initial launch but also doesn't need to be tied to phase two.

We could also have a "phase zero" that consists of any improvements we want to make to the existing resources before we're able to create the unified site.

## Note

The site needs a real name! We don't want to refer to it by its URL forever. The name "OpenDataPolicies.org" was just a temporary measure to avoid Open Data Policies Decoded, since the site no longer runs on the State Decoded platform.

**Comment by SLarrick** [comment]
Love it. Will add more thoughts later today. Quick note on design: as long as we are funded by What Works Cities in doing this work (the foreseeable future) we should use WWC Styleguide/Design templates, combined with existing Sunlight styleguides. This means something aesthetically similar to the Public Policy for Public Data Checklist.

**Comment by SLarrick** [comment]
In terms of a name, it should be something that conveys that this is a "one stop" shop for everything one might need related to local open data policy research, comparison, writing, etc.

Some quick ideas:

- "open data policy hub"
- "open data policy help desk"
- "open data policy one stop"
- "public policy for public data - resource center"

None of these are very sexy so we should talk further with the team.

**Comment by gregjd** [comment]
I agree with your first comment (it also means less stuff to figure out!).

From your second comment, I like "Open Data Policy Hub" the best, though I still think it doesn't quite hit the nail on the head — but I don't have a better suggestion and in any case we will discuss further with the team.
